README.md

```diff
@@ -1,56 +1,74 @@
 # ![JMP](logo.png)
 
-## Getting Started
+# Table of Contents:
+- [JMP](#jmplogopng)
+- [Getting Started](#getting-started)
+  * [Prerequisites](#prerequisites)
+  * [Installation](#installation)
+  * [Development](#development)
+    + [Frontend](#frontend)
+    + [Backend](#backend)
+- [Authors](#authors)
+- [License](#license)
 
-### Pre-requisites
+<small><i><a href='http://ecotrust-canada.github.io/markdown-toc/'>Table of contents generated with markdown-toc</a></i></small>
 
-Make sure that you have docker, npm and composer installed on your machine.
+# Getting Started
 
-### Installation
+## Prerequisites
 
-#### Frontend
+Make sure you have [docker](https://www.docker.com/) and [npm](https://www.npmjs.com/) installed on your machine.
 
-Everything related to the frontend is located in [vue](vue). The frontend uses **[Vue.js](https://vuejs.org/)**.
+## Installation
 
-> Install Dependencies
+Clone the repository
+```bash
+git clone https://github.com/Simai00/jmp.git
+cd jmp
+```
 
-In the vue folder:
+Build vue
 ```bash
+cd vue
 npm install
+npm run build
 ```
+> To run vue at development, read this [doc](vue/README.md)
 
-#### Backend
-
-Everything related to the backend is located in [api](api). The backend uses the **[Slim Framework](https://www.slimframework.com/)**.
-
-> Install Dependencies
+Create file `api/.env` and set all variables declared in [.env.example](api/.env.example)  
+If you don't create the file by yourself, one will be created with the default values. **This is not recommended in production!**   
+> Read more about all variables at [dotenv](docs/dotenv.md)
 
-In the projectfolder:
+Build and start the docker containers
 ```bash
-composer install -d api
+docker-compose up -d
 ```
 
-### Docker
+Install all the dependencies using the composer of the app container
+````bash
+docker exec -it app composer install
```

Comment:
I think this step is not necessary as the app container installs all composer dependencies when the app container is built.

Reply:
Yes, it's necessary. I've just tested it again.